src/main/java/io/botcrafting/botcraft/configuration/constant/MessageConstant.java

```diff
@@ -15,6 +15,9 @@
             "Para buscar um livro você precisa digitar o que deseja encontrar.\n" +
             "Por exemplo: \"/livro Os Miseráveis\".\n\n" +
             "Tente novamente, por favor.";
+    public static String FOR_THE_HORDE = "pela horda";
```

Comment:
Just commenting for the record: We should move these constants from this file closer to where they are used and maybe move some to configurable properties.